Source/control.h

```diff
@@ -32,6 +33,15 @@ namespace devilution {
 #define SPANEL_WIDTH 320
 #define SPANEL_HEIGHT 352
 
+enum class ControlTypes : uint8_t {
+	None,
+	KeyboardAndMouse,

```

Comment:
Should keyboard be considered separate from mouse? Maybe someone thinks that touch + keyboard is amazing?

Reply:
Devices like the new RG552 have a physical Gamepad + a touch screen

Reply:
This is the same for Switch (and there are also some Android devices). 3DS and Vita also has a similar situation except the touch surface isn't directly mapped to the screen. There is nothing about this change that should cause issues for such platforms compared to previously. But we might want to do add individual enhancements to these platforms to take additional advantage of there unique interface, similarly to what is currently done for Vita (something I'm not currently taking in to account and why this is a draft).

Reply:
The vita unique touch handling is now taken in to account.